src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -217,15 +216,9 @@ export class RbdFormComponent extends CdForm implements OnInit {
     this.rbdForm.get('obj_size').disable();
     this.rbdForm.get('stripingUnit').disable();
     this.rbdForm.get('stripingCount').disable();
-
-    /* RBD Image Format v1 */
-    this.rbdImage.subscribe((image: RbdModel) => {
-      if (image.image_format === RBDImageFormat.V1) {
-        this.rbdForm.get('deep-flatten').disable();
-        this.rbdForm.get('layering').disable();
-        this.rbdForm.get('exclusive-lock').disable();
-      }
-    });
+    this.rbdForm.get('deep-flatten').disable();
```

Comment:
This doesn't seem right: `deep-flatten` feature can be disabled (but can't be re-enabled later).

Reply:
Yes, `deep-flatten` can't be enabled after the image is created.